Gruntfile.js

```diff
@@ -0,0 +1,58 @@
+const path = require('path');
```

Comment:
i'm curious if this will work, because i'm not sure if our grunt file is being transpiled by Babel. If it doesn't work, don't worry about it. The gruntfile could be an exception.